code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -459,3 +459,50 @@
 	if(prob(20))
 		M.heal_bodypart_damage(3,1)
 	..()
+
+
+////Lavaland Flora Reagents////
+
+
+/datum/reagent/consumable/entpoly
+	name = "Entropic Polypnium"
+	id = "entpoly"
+	description = "An ichor, derived from a certain mushroom, makes for a bad time."
+	color = "#1d043d"
+/datum/reagent/consumable/entpoly/on_mob_life(mob/living/M)
+	if(current_cycle >= 10)
+		M.Paralyse(2, 0)
+		. = 1
+	if(prob(20))
+		M.losebreath += 4
+	M.adjustBrainLoss(2*REM)
+	M.adjustToxLoss(3*REM,0)
+	M.adjustStaminaLoss(10*REM,0)
+	M.blur_eyes(5)
+	..()
+
+/datum/reagent/consumable/tinlux
+	name = "Tinea Luxor"
+	id = "tinlux"
+	description = "A stimulating ichor which can make the user resistant to being subued. "
```

Comment:
Can we please stop adding anti-stun to mining?

There is all of one stun on mining. Stuns basically mean nothing to mining combat, but can adversely affect station combat significantly. Change this, please.